### DISCUSSION TITLE
Load future games/matchups in addition to the next one

### DISCUSSION THREAD

**Discussion by AlexHawkins1** [vasqued2/ha-teamtracker#114] [Ideas]
I would like the ability to load the next x number of events. For Example in UFC the ability to load the full next card and not just the next individual fight

**Comment by vasqued2** [comment]
Unfortunately, this is a limitation of the API.  ESPN populates the API so many days in advance of the event.  This varies based on the sport but generally it only goes far enough into the future to ever return a single event.

Even if I could pull back multiple events, it would be a breaking change to return a list of multiple events instead of a single one.  This would not only break the card but also any automations or other custom use cases that anyone had created.  For this reason, even if ESPN suddenly started to return more events, I would likely still only return one so I wouldn't break everyone's existing setup.

[This article](https://smarthomepursuits.com/teamtracker-integration-home-assistant-real-time-sports-scores/?expand_article=1) shows an example of how you might be able to use a Google calendar integration to show upcoming games.
